src/v1/texture.rs

```diff
@@ -15,64 +15,64 @@ enum_number!(
         NearestMipmapLinear = 9986,
         LinearMipmapLinear = 9987,
     }
-);
+}
 
 impl Default for Filter {
     fn default() -> Filter {
         Filter::Nearest
     }
 }
 
-enum_number!(
+enum_number! {
     Wrap {
         Repeat = 10497,
         ClampToEdge = 33071,
         MirroredRepeat = 33648,
     }
-);
+}
 
 impl Default for Wrap {
     fn default() -> Wrap {
         Wrap::Repeat
     }
 }
 
-enum_number!(
+enum_number! {
     Format {
         Alpha = 6406,
         Rgb = 6407,
         Rgba = 6408,
         Luminance = 6409,
         LuminanceAlpha = 6410,
     }
-);
+}
 
 impl Default for Format {
     fn default() -> Format {
         Format::Rgba
     }
 }
 
-enum_number!(
+enum_number! {
     Target {
         Texture2d = 3553,
     }
-);
+}
 
 impl Default for Target {
     fn default() -> Target {
         Target::Texture2d
     }
 }
 
-enum_number!(
+enum_number! {
     TexelType {
```

Comment:
Another alternative might be
```rust
TexelType {
    UnsignedByte = 5121,
    UnsignedShortR5G6B5 = 33635,
    UnsignedShortR4G4B4A4 = 32819,
    UnsignedShortR5G5B5A1 = 32820,
 }
```
It's quite awkward that `UnsignedByte` is by itself when really that means `UnsignedByteR8B8G8A8`. On second thoughts (and to avoid the [bikeshed effect](https://en.wikipedia.org/wiki/Law_of_triviality)) I think your proposal will be best.